test/http.js

```diff
@@ -360,64 +359,55 @@ describe('http', () => {
     test(
       'should serialize fetch-like response and call serializeHeaders',
       () => {
-        const headers = {
-          Authorization: ['Basic hoop-la', 'Advanced hoop-la']
-        }
-
-        const res = fetchMock.mock('http://swagger.io', {headers})
+        const response = new Response('data', {
+          status: 200,
+          headers: {
+            Authorization: 'Basic hoop-la, Advanced hoop-la',
+            'Content-Type': 'text/plain',
+          }
+        })
 
-        return fetch('http://swagger.io').then((_res) => { // eslint-disable-line no-undef
-          return serializeRes(_res, 'https://swagger.io')
-        }).then((resSerialize) => {
-          expect(resSerialize.headers).toEqual({authorization: ['Basic hoop-la', 'Advanced hoop-la']})
-        }).then(fetchMock.restore)
+        return serializeRes(response, 'https://swagger.io')
```

Comment:
keep the eslint-disable comment.

Reply:
No need for disable comment here. `serializeRes` is explicitly imported symbol in global scope.